reddit_experiments/__init__.py

```diff
@@ -132,7 +134,7 @@ def _get_experiment(self, name: str) -> Optional[Experiment]:
             self._cfg_data = self._get_config()
 
         if name not in self._cfg_data:
-            logger.info("Experiment <%r> not found in experiment config", name)
+            warnings.warn(f"Experiment <{repr(name)}> not found in experiment config", RuntimeWarning)
```

Comment:
The docs say `RuntimeWarning` is "Base category for warnings about dubious runtime features." I think that sounds like it should be for interpreter-level stuff, not our use. It'd probably be fine to leave this un-specified (which defaults to`UserWarning`) or subclass our own ExperimentsWarning if we really want to be fancy.